src/model.js

```diff
@@ -152,6 +152,16 @@ export default class MediaPlayerObject {
       && PROGRESS_PROPS.every(prop => prop in this.attr);
   }
 
+  get hasPrev() {
```

Comment:
We should name these similar to the other functions that check for supported features:
https://github.com/kalkih/mini-media-player/blob/8f5a9ab87ce9bbeb39960b23651f0e5d3d425ed4/src/model.js#L200-L214
Prefixed with `supports` instead of `has`.